src/components/EnlargedPreview/index.js

```diff
@@ -30,12 +33,18 @@ class EnlargedPreview extends Component<Props, State> {
   }
 
   componentWillUpdate(nextProps: Props, nextState: State) {
+    if (nextState.isExpanded !== nextProps.isNavigationDisabled) {
+      this.props.setNavigationDisabled(nextState.isExpanded)
+    }
     if (nextState.isExpanded !== nextProps.isFullScreen) {
       this.props.setFullScreen(nextState.isExpanded)
     }
   }
 
   componentWillUnmount() {
+    if (this.props.isNavigationDisabled) {
```

Comment:
same here. not sure if it's really necessary to prevent setting something redundantly

Reply:
OK, sure, I'll change that to _hard off_.